profile.yaml

```diff
@@ -12,3 +12,9 @@ spec:
     - name: nginx-deployment
       path: nginx/deployment
       kind: Kustomize
+    - name: nginx
+      kind: HelmChart
+      chart:
+        helm_url: https://charts.bitnami.com/bitnami
+        helm_chart: nginx
+        helm_chart_version: "8.8.1"
```

Comment:
do we want to deploy a different chart? Atm if you were to deploy this profile it would deploy the exact same chart twice, once via the local chart and once for the remote.

Reply:
Yeah, I thought so too. I'll choose something else.

Reply:
shall we rename this repo if the new helm chart isn't a nginx chart?

Reply:
I was thinking about that. Yeah that makes sense if we now mix these things. Something like, example-profile.